apps/ensrainbow/src/lib/api.ts

```diff
@@ -42,7 +47,11 @@ export async function createApi(db: ENSRainbowDB): Promise<Hono> {
     logger.debug("Label count request");
     const result = await server.labelCount();
     logger.debug(`Count result:`, result);
-    return c.json(result, result.errorCode);
+
+    // Map error codes > 1000 to 500, otherwise use the original code
+    const statusCode = result.errorCode && result.errorCode >= 1000 ? 500 : result.errorCode;
```

Comment:
Can `result.errorCode >= 100` be ever `true`? We haven't changed anything int the server code, so it won't know about new error codes.